Cakes-Co/css/style.css

```diff
@@ -0,0 +1,306 @@
+/*  Add your styling here */
```

Comment:
I'm not quite sure why this is in this pull request, if you make changes to cakes Co, do it on the branch you originally pushed Cakes-Co on please so it doesn't get confused